### ISSUE TITLE
How to add scheduled_date

### ISSUE THREAD

**Issue by laushkeen** [shelomentsevd/telegram-api#31]
Telegram has the functionality to schedule messages but I don't know how to add it

https://core.telegram.org/api/scheduled-messages

**Comment by savely-krasovsky** [comment]
@laushkeen it's currently better to use this: https://github.com/gotd/td

**Comment by laushkeen** [comment]
@L11R Yes, I know but I need a relatively small thing to do. I don't want to learn tons of methods. Ideally, I want a lib that can authorize me as a user and provide a method to schedule a message to a certain peer. That's all I need
